### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by ghost** [redsudo/RedSea#46]
Given this Album url: https://tidal.com/browse/album/17172679

When downloading

The following error is encountered:
```
Traceback (most recent call last):
  File "C:\dev\redsea\redsea.py", line 225, in <module>
    main()
  File "C:\dev\redsea\redsea.py", line 169, in main
    md.download_media(track, preset['quality'], media_info)
  File "C:\dev\redsea\redsea\mediadownloader.py", line 177, in download_media
    temp_file = self._dl_url(stream_data['url'], track_path)
  File "C:\dev\redsea\redsea\mediadownloader.py", line 38, in _dl_url
    with open(where, 'wb') as f:
FileNotFoundError: [Errno 2] No such file or directory: './music/deadmau5 -  Album Title Goes Here \\01 - Superliminal.flac'
```

I am using the changes from https://github.com/redsudo/RedSea/pull/43 (to support the new URLs)

I have made the following changes in the _sanitise_name method:
```
   def _sanitise_name(self, value):
        value = str(value)
        value = unicodedata.normalize('NFKD', value).encode('ascii', 'ignore').decode('ascii')
        value = re.sub(r'[^\w\s-]', '', value).strip()
        return re.sub(r'[-\s]+', ' ', value)
```
My the changes are based on the "slugify" method located in https://github.com/django/django/blob/master/django/utils/text.py

After my changes
```
<<< Downloading single track... >>>
=== Downloading track ID 17172680 ===
        Track: 01
        Title: Superliminal
        Artist: deadmau5
        Album: > Album Title Goes Here <
        ----
        Grabbing album info...
        Grabbing stream URL...
        Download progress: 100%
        Looks like file is encrypted. Decrypting...
        Downloading album art...
        Download progress: 101%
        Tagging media file...
=== 1/1 complete (100% done) ===

> Download queue: 1/1 items complete (100% done) <

> All downloads completed. <
```

**Comment by redsudo** [comment]
Fixed in 2.2

**Comment by firebfm** [comment]
I cannot download any track with asian filenames. FileNotFoundError: [Errno 2] No such file or directory. My old version of redsea back in May is still working fine. Please revert the change.
